online_meetings/canvas_zoom_meetings.py

```diff
@@ -168,12 +150,12 @@ def zoom_course_report(
         add_course_ids: list = None
```

Comment:
I probably should have updated this in my PR. Could we make the type hint here `Union[List[int], None]`?

Reply:
Or `Sequence` is probably fine here.

Reply:
Sequence[int] has no attribute remove. Needs to be a List it looks like but I can update it.